CONTRIBUTING.md

```diff
@@ -34,9 +34,9 @@ First of all, thank you for contributing to Meilisearch! The goal of this docume
 
 To run this project, you will need:
 
-- Node.js >= v14 and node <= 18
+- Node.js >= v16 and node <= 18
 - Yarn
-
+- vitest
```

Comment:
While it might run on Node.js 16, we're not testing it. And we shouldn't as it's End Of Life.
`vitest` shouldn't be required, `yarn` installs it, it's part of the `devDependencies`.

I guess it's fine for now, I might tweak it in the future. Also Node.js 18 EOL is approaching, when it arrives, I'll change this.